src/app/sass/main.scss

```diff
@@ -80,9 +80,26 @@ html, body, #root, #root > div , #root > div > div {
 
 #overDiffMap {
     position: absolute;
-    top: 136px;
+    top: 50%;
     left: 0px;
     width: 100%;
     max-height: calc(100% - 112px);
     z-index: 1000;
 }
+
+#overDiffMap, .slider, .slider > div {
+    pointer-events: none;
+}
+
+.slider > div > div {
+    pointer-events: auto;
+}
+
+.slider > div > div > div {
+    pointer-events: auto;
+    background-image: url('https://www.shareicon.net/data/512x512/2015/10/06/113433_arrows_512x512.png');
```

Comment:
Can you verify that this icon is in the public domain? I took a look at the site, and saw some that are creative commons attribution, etc. I couldn't find this particular icon. Also, I think we should download this and serve it locally.